detection-rules/attachment_office_file_relationship_cred_theft.yml

```
@@ -0,0 +1,36 @@
+name: "Attachment: Office file contains OLE relationship to credential phishing page"
+description: |
+  Office file OLE relationship link is a credential page, or contains credential phishing language.
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments,
+          (
+            .file_extension in~ $file_extensions_macros
+            or (
+              .file_extension is null
+              and .file_type == "unknown"
+              and .content_type == "application/octet-stream"
+              and .size < 100000000
+            )
+          )
+          and any(file.oletools(.).relationships,
+                  any(ml.nlu_classifier(beta.linkanalysis(.target_url).final_dom.inner_text).intents,
```

this one is much more friendly to work with, especially for NLU
```suggestion
                  any(ml.nlu_classifier(beta.linkanalysis(.target_url).final_dom.display_text).intents,
```

---

Thank you!